functions/check-started/index.js

```diff
@@ -1,11 +1,46 @@
 'use strict'
 
-const uuid = require('uuid/v4')
+const azureStorage = require('azure-storage')
+const bluebird = require('bluebird')
 const moment = require('moment')
+const sqlService = require('less-tedious')
+const uuid = require('uuid/v4')
+const winston = require('winston')
+const { TYPES } = require('tedious')
+
+winston.level = 'error'
+const config = require('../config')
+sqlService.initialise(config)
+
+const checkStatusTable = '[checkStatus]'
+const checkTable = '[checkFormAllocation]'
+const schema = ['mtc_admin']
+const preparedCheckTable = 'preparedCheck'
+let azureTableService
+initAzureTableService()
 
-module.exports = function (context, checkStartMessage) {
+module.exports = async function (context, checkStartMessage) {
   context.log('check-started message received', checkStartMessage.checkCode)
-  // TODO: purpose: process check start messages and put into pupilEvents table
+
+  // Update the admin database to update the check status to Check Started
+  try {
+    await updateAdminDatabaseForCheckStarted(checkStartMessage.checkCode, context.log)
+    context.log('SUCCESS: Admin DB updated')
+  } catch (error) {
+    context.log.error(`ERROR: unable to update admin db for [${checkStartMessage.checkCode}]`)
+    throw error
+  }
+
+  // Delete the row in the preparedCheck table - prevent pupils logging in again.
+  try {
+    await deleteFromPreparedCheckTableStorage(checkStartMessage.checkCode, context.log)
+    context.log('SUCCESS: Table storage cache deleted from preparedCheck table')

```

Comment:
can we change this message to `SUCCESS: pupil check row deleted from preparedCheck table` please

Reply:
Yes, that works for me